README.md

```diff
@@ -1,63 +1,57 @@
 # Talos
-A multi level authorization project
+A file system permission project.
 
 ## The concept
-The authorization in this project is an attempt to conveniently define what users can access what resources. Resources are uniquely identified using a "resource identifier", which is a string, possibly consisting of multiple levels separated by slashes in order to keep them well-organized. The users in the systeam each have a role, which also can inherit permissions of another role. The permissions then can be specified using so-called permission rules. For example: 
+This project aims to provide a convenient description mechanism for file system permissions. Resources are identified using a string consisting of one or more levels separated by slashes. Users of the system each have a role which can inherit permissions of other roles. Permissions are specified using so-called permission rules, which are triplets of a boolean (allow or deny), a role and a resource. For example: 
 
 ```
-Admin > User
+Student > Mara
+Student > Jeffrey
 
-allow User  /
-deny  User  /profile
-allow User  /profile/[id]
-allow Admin /profile
-deny  Admin /profile/*/password
-allow Admin /profile/[id]/password
+allow Admin   /
+allow Student /home/[id]
+allow Mara    /srv/nfs/music
+deny  Jeffrey /home/[id]/config
+deny  Admin   /home/*/personalsecrets
 ```
 
-These are the access rules for a website where people have a profile and everyone can access their own profile, but administrators can also access other people's profiles, except for the password sections.
+## The tools
 
-## Examples
-Here are some examples of valid sets of access rules.
+### Inheritance
+```
+A > B
+B > C
+
+allow A x
+deny  A x/*
+allow B x/y
+allow C x/z
+```
+`A` has access to `x`, `B` has access to both `x` and `x/y` and `C` has access to `x`, `x/y` and `x/z`.
 
-### Multiple inheritances
 ```
-B > A
-C > B
+A > B
 
-allow A a
-deny  A a/*
-allow B a/b
-allow C a/c
+allow A x/*
+deny  B x/y
 ```
-In this case, A has access to a, B has access to both a and a/b and C has access to a, a/b and a/c.
+Although `A` has access to `x` and `x/y`, `B` does not have access to `x/y` as inherited access is overridden.
 
 ### Sets
 ```
-Admin > User
+User > Admin
 
-allow User  devices
+allow Admin devices
```

Comment:
This should be `allow User devices`, right? Since `User` is a superset of `Admin`, the former stays when merged with the latter one.